### DISCUSSION TITLE
How does include and exclude work

### DISCUSSION THREAD

**Discussion by nikhil-rajan-neu** [Azure-Player/azure.datafactory.tools#109] [Q&A] [answered]
so i assume if you dont inlude / exclude anyting .. it deploys everything by default
if specify an item as exclude , it deploys everything but that the item in exclude 
if specify exclude and include , does it only include the one in the list ?

**Comment by NowinskiK** [comment, marked as answer]
From the documentation:

> Bear in mind that *Includes* and *Excludes* lists are **rules out each other**.  
Objects would be excluded from deployment only if *Includes* list remains empty.  
When both lists are empty - all objects going to be published.

https://github.com/SQLPlayer/azure.datafactory.tools#publish-options
